server/src/main/java/org/apache/cassandra/sidecar/modules/SidecarSchemaModule.java

```diff
@@ -50,13 +55,16 @@ public SidecarSchema sidecarSchema(Vertx vertx,
         SidecarInternalKeyspace sidecarInternalKeyspace = new SidecarInternalKeyspace(configuration);
         // register table schema when enabled
         resolver.resolve().values().forEach(tableSchema -> {
+            LOGGER.info("Registering table schema: {}", tableSchema);
             try
             {
                 sidecarInternalKeyspace.registerTableSchema(tableSchema);
             }
             catch (Throwable cause)
             {
-                throw new RuntimeException("Failed to register table schema: " + tableSchema, cause);
+                String message = "Failed to register table schema: " + tableSchema;
+                LOGGER.error(message);
+                throw new RuntimeException(message, cause);
```

Comment:
The same error message should be printed by guice when bootstraping. But not against having another error log message here.